include/seqan3/alignment/configuration/align_config_edit.hpp

```diff
@@ -49,7 +48,6 @@ namespace seqan3::align_cfg
  * trigger the slower algorithm which can handle the case if the ends are free in the second sequence instead of the
  * first sequence.
  */
-inline constexpr configuration edit = method_global | scoring{nucleotide_scoring_scheme{}} |
-                                      gap{gap_scheme{gap_score{-1}}};
+inline constexpr configuration edit = scoring{nucleotide_scoring_scheme{}} | gap{gap_scheme{gap_score{-1}}};
```

Comment:
I thought we wanted to call it `edit_scheme`?

Reply:
Damn, this got lost in all the other stuff :+1: